### ISSUE TITLE
Focus cell should always come with an underlying selection

### ISSUE THREAD

**Issue by llorca** [palantir/blueprint#1072]
### Bug report

- __Package version(s)__: table 1.13.0

### Description

When a table initially loads with a focus cell, the focus cell is in A1 but is missing its underlying selection.

We could go two ways about this:
1. Show the focus cell with the cell below selected.
1. Don't show the focus cell on initial load, the focus cell will then come after the first selection is made.

No strong opinion on either of these yet, though 1. seems to be the default in both Excel and Spreadsheet.

**Comment by giladgray** [comment]
is this really a bug? i consider focus and selection as two distinct things (and the table does too, implementation-wise).

**Comment by gscshoyru** [comment]
I think it's a bug -- focus and selection are actually quite entangled. If we have focus turned on, there should always be at least one cell selected -- the focused cell is always selected.

**Comment by llorca** [comment]
Yep they are distinct things. A selection coming without a focus cell is one thing, but a focus cell coming without a selection? That doesn't make a lot of sense, does it? Also, this only happens on table load, so clearly seems like an oversight

**Comment by gscshoyru** [comment]
Yeah, it was an oversight -- my bad -- should be reasonably straightforward fix actually. 

Technically you can also get this state via deselection of single cells right now, but since we want to turn that off we should just need to take care of the loading case.

**Comment by tgreenwatts** [comment]
I still want to have my table (table, not spreadsheet, just 1 small part of an app) and not already have something selected - for visual purposes. It's distracting unless I'm specifically trying to interact with the table

**Comment by gscshoyru** [comment]
It shouldn't be distracting in any sort of way -- if you have a focus cell, the area under the focus cell should be selected. It's only on start that this is currently broken -- if you say, use the arrow keys to move the focus around, the selection is already following along. It's just the initialization behavior that needs fixing, and it'll bring it in line with the already existing behavior. From a visual standpoint, it'll just shade the focused cell slightly darker -- the way it looks if you arrow to the left and back right now.

**Comment by cmslewis** [comment]
When we polish this further, my personal vote is to not have the selection background behind the focus cell. Other spreadsheet products seem to follow that approach.

**Comment by tgreenwatts** [comment]
Ok so talked offline to @llorca about this and I understand better now - we *already* have a focus cell on load by default, it just doesn't have a selection with it. 
Options are:
1. No focus cell by default
2. Focus cell (with selection) in A1 

Option 2 works for spreadsheets better. We can reexamine making this an option for apps when we're better suited to support spreadsheets, for now I propose we go with option 1.

**Comment by gscshoyru** [comment]
We already are doing 1 (which is totally correct). We should 2 if and only if focus cell is enabled -- which is what's presently missing. In other words -- on initialization, check if focus cell is true, and if it is, stick a selection underneath it. That's what this ticket is doing. We certainly don't want to do it if focus is not enabled.

**Comment by llorca** [comment]
@gscshoyru the two options from Taryn are proposals for when the focus cell is turned on. She proposes we do 1. when the focus cell is enabled, which we don't do at the moment.
